src/connection-manager/manifests.ts

```diff
@@ -85,54 +79,24 @@ export async function handleManifest(
 export async function updateUNLManifests(): Promise<void> {
   try {
     log.info('Fetching UNL...')
-    const unl: UNLBlob = await fetchValidatorList(await getFirstUNL('main'))
-    const promises: Array<Promise<void>> = []
-
-    unl.validators.forEach((validator: UNLValidator) => {
-      const manifestHex = Buffer.from(validator.manifest, 'base64')
-        .toString('hex')
-        .toUpperCase()
-      promises.push(handleManifest(manifestHex))
+    const networks = await getNetworks()
+    networks.forEach(async (network) => {
+      const unl: UNLBlob = await fetchValidatorList(network.unls[0])
+      const promises: Array<Promise<void>> = []
+
+      unl.validators.forEach((validator: UNLValidator) => {
+        const manifestHex = Buffer.from(validator.manifest, 'base64')
+          .toString('hex')
+          .toUpperCase()
+        promises.push(handleManifest(manifestHex))
+      })
+      await Promise.all(promises)
     })
-    await Promise.all(promises)
   } catch (err) {
     log.error('Error updating UNL manifests', err)
   }
 }
 
-/**
- * This function loops through all signing keys in the validators table and queries rippled
- * to find the most recent manifest available.
- *
- * @returns A promise that resolves to void once all of the latest manifests have been saved.
- */
-export async function updateManifestsFromRippled(): Promise<void> {
```

Comment:
This might still be useful to do for mainnet

Reply:
Validators on mainnet is updated with xl.ripple.com unl. If you want to keep those unused ones then sure I can keep this function.

Reply:
I'm thinking that it would be useful for the case where a validator's manifest changes but the UNL hasn't been updated yet.

Reply:
If you look here: https://livenet.xrpl.org/network/validators the domain is available for a bunch of validators. If we just pull from the UNL, we won't have the domains for the other validators. This makes it hard to e.g. identify validator operators that need to upgrade

Reply:
I see. Will restore the functionality then.